.github/workflows/test.yml

```diff
@@ -25,5 +25,11 @@ jobs:
       - name: Install dependencies
         run: yarn
 
-      - name: Run the tests
+      - name: Run the unit tests
         run: yarn test
+
+      - name: Cypress run
```

Comment:
@tanvirchahal let's call it end to end tests or integration tests so it's more accurate

Reply:
@anuveyatsu changed the name to "Run the integration tests"